xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/displayers/BaseDisplayer.vue

```diff
@@ -91,55 +95,127 @@ export default {
   // Add the displayerMixin to get access to all the displayers methods and computed properties inside this component
   mixins: [displayerMixin],
 
+  components: {
+    XWikiLoader,
+  },
 
   props: {
     viewOnly: {
       type: Boolean,
       default: false,
     },
-  },
-
-  data () {
-    return {
-      // Whether the displayer is in view or edit mode
-      isView: true,
-    };
+    isView: {
+      type: Boolean,
+      default: true,
+    },
+    isLoading: {
+      type: Boolean,
+      default: false
+    }
   },
 
   computed: {
-    isEditable () {
-      return this.logic.isEditable({
+    // Checks if the displayed is allowed to be edited and if the livedata is in a state where this displayer can be
+    // edited.
```

Comment:
Could you improve this comment. We don't edit the displayer. We edit the value of a property.